typescript/cli/src/config/warp.ts

```diff
@@ -278,11 +279,25 @@ export async function createWarpRouteDeployConfig({
         tokenMetadata?.symbol,
         'Error deriving token metadata, please check the provided token addresses',
       );
+
+      const symbol = tokenMetadata.symbol;
+      let warpRouteId;
+      if (!context.skipConfirmation) {
+        warpRouteId = await getWarpRouteIdFromWarpDeployConfig(
+          context.registry,
+          warpRouteDeployConfig,
+          symbol,
+        );
+      }
+
```

Comment:
Can we add a test for this new prompting here?

Reply:
Added